tox.ini

```diff
@@ -7,7 +7,7 @@ deps=
     -rrequirements/compiled.txt
     -rrequirements/dev.txt
 commands =
-    coverage run manage.py test --noinput -v2 kuma
+    coverage run manage.py test --noinput -v2 {posargs:kuma}
```

Comment:
Nah, git will auto merge that since there is nothing else in the other branch changing on that line.